schema.d.ts

```diff
@@ -372,6 +374,29 @@ export interface CssPart {
   deprecated?: boolean | string;
 }
 
+/**
+ * The description of a CSS Custom State
```

Comment:
We don't do this a lot yet, but I think it'd be helpful here to link to MDN: https://developer.mozilla.org/en-US/docs/Web/CSS/:state

Reply:
Linked to https://developer.mozilla.org/en-US/docs/Web/API/CustomStateSet instead as that has a bit more of a high level overview imo.